### PR TITLE
Add new whitespace check to checkstyle rules

### PR DESCRIPTION
**Pull request by cco3**
Specifically, look for instances of "){".

### CODE REVIEW

File: android/PhysicalWeb/app/config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -39,6 +39,11 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <property name="message" value='Get rid of trailing whitespace.' />
   </module>
 
+  <!--module name="RegexpSingleline">
+    <property name="format" value="a" />
+    <property name="message" value='Get rid of trailing whitespace.' />
+  </module-->
```
Comment (hayesjordan) [inline android/PhysicalWeb/app/config/checkstyle/checkstyle.xml]: Is there a reason this is commented out?

Comment (cco3) [inline android/PhysicalWeb/app/config/checkstyle/checkstyle.xml]: good catch

**Comment by hayesjordan** [comment]
LGTM

**Comment by hayesjordan** [comment]
Can you fix the merge conflict?

**Pull request merged**